index.html

```diff
@@ -5,12 +5,85 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Caveat|Courgette|Lobster&display=swap" rel="stylesheet">
     <title>Responsive Cake webpage</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+ <main>
+   <header>
+    <div class="Logo">
+        <img class="logoup" src="Yusuf Logo.png" alt="logo"/>
+        <p>best home made cake delivered at your door</p>
+     </div>
+     <h3>Add To Cart</h3>
+     <div>
+        <ul>
+            <li><a class="first-menu" href="#">Menu</a>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Cakes</a></li>
+                    <li><a href="#">Lesson</a></li>
+                    <li><a href="#">Ordering</a></li>
+                    <li><a href="#">Download recipes</a></li>
+                    <li><a href="#">About</a></li>
+                </ul>
+            </li>
+        </ul>
+    </div>
+  </header>
+<table>
+
+    <tr>
+        <th>Our Cakes</th>
+    </tr>
+    
+
+    <tr>
+        <td><a href="images/cake-wedding.jpg"target="_blank"><img src="images/cake-wedding.jpg"></a></td>
+        <td><a href="images/cake2.jpg"target="_blank"><img src="images/cake2.jpg"></td>
+        <td><a href="images/cake3.jpg" target="_blank"><img src="images/cake3.jpg"></td>
+    </tr>
+    <tr>
+        <td><a href="images/cake4.jpg" target="_blank"><img src="images/cake4.jpg"></a></td>
+        <td><a href="images/cake5.jpg" target="_blank"><img src="images/cake5.jpg"></a></td>
+        <td><a href="images/cake6.jpg" target="_blank"><img src="images/cake6.jpg"></a></td>
+    </tr>
+</table>
+<p>Our Little Story</p>
+<p>As well as our signature Party Cap Cakes hand baked by us in an ever changing selection of the latest and greatest flavours know to man and woman! we do lots of other tasty stuff too....<br>
+
+We can talk a long time about the things we love about Cake Stories, such as the beautiful coffee, the huge selection of loose leaf tea, the fact all our food, scones and cakes are made by us, our super friendly and brilliant team, the good hearty freshly prepared food or the many, many lights but the best thing we can do is to invite you for a visit... the door is open!</p>
+<p>See us on Channel</p>
+<div class="iframe-video">
+    
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/hZux_BvIA08" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+ </main>
+ <div>"
```

Comment:
You shouldn't have any elements that don't live in either your header, main or footer elements.